### ISSUE TITLE
Inline image float is doing something weird with ad

### ISSUE THREAD

**Issue by phillipadsmith** [TheTyee/design-article.thetyee.ca#195]
<img width="574" alt="mass_sea_star_deaths_shake_up_marine_food_chain___the_tyee" src="https://cloud.githubusercontent.com/assets/34128/16399470/8443d520-3c87-11e6-9329-47fde9c563bc.png">

**Comment by alexgreen** [comment]
Is there a link for this page?

**Comment by phillipadsmith** [comment]
> Is there a link for this page?

No, it was a test story (now deleted).

**Comment by alexgreen** [comment]
So the problem occurred when an figure containing an image immediately preceded an aside containing an ad? And it appears to push the ad img out of position?

**Comment by phillipadsmith** [comment]
> So the problem occurred when an figure containing an image immediately preceded an aside containing an ad? And it appears to push the ad img out of position?

Correct.

**Comment by alexgreen** [comment]
We can create another test page to reproduce the issue

**Comment by phillipadsmith** [comment]
> We can create another test page to reproduce the issue

Yes, can do.

**Comment by alexgreen** [comment]
I had originally floated the ad-box left to avoid these collisions.
@MrBryan you disabled this float, commenting that it was causing "ads to 'clump' sometimes and to break up paragraphs". Can you show some examples of this?
In the meantime I've re-enable the float ( a809069c20cd1edbb71f6829074631ac7c0d7293 ) and it does seem to solve the collision issues.

**Comment by MrBryan** [comment]
yes it was causing the two ads to show one after the other on some pages.. or rather disabling it fixed that issue which may better be addressed some other way at least.

**Comment by MrBryan** [comment]
and by break up paragraphs I meant that sometimes ads would show in the middle of a paragraph rather than between two. I'll enable it temporarily and find an example on preview before I pull/merge anything to the bric-mod branch

**Comment by MrBryan** [comment]
Here's a paragraph broken example if you see this in the next little while i'll leave it there or if I have already disabled that float you can enable in web inspector to see the effect (in this case the ad still ends up in an awkward spot with float disabled but its because the heading is a separate paragraph so it's technically working right)  

http://preview.thetyee.ca/Culture/2016/07/01/Four-Forgotten-BC-Heritage-Moments/

**Comment by MrBryan** [comment]
![image](https://cloud.githubusercontent.com/assets/1721891/16543538/ffbb7140-40a6-11e6-879e-d71799f85c28.png)

**Comment by MrBryan** [comment]
http://preview.thetyee.ca/Opinion/2016/06/23/Young-Workers-Should-Celebrate-CPP-Changes/ another one

**Comment by alexgreen** [comment]
Looks like this problem is back. (ಠ_ಠ)
http://preview.thetyee.ca/Opinion/2016/08/01/Vancouver-is-Home/index.html

The attached image shows a top border from the ad-box--bigbox that is appearing above the preceding image. As weeks ago the problem is that the left float applied to ad-box has been disabled from the ad-box--bigbox. It happens here in line 1722 of the css (which is my addition):

```
@media screen and (min-width: 976px) {
    /* This removes the float from bigbox ads so that they don't get pushed out of position by asides. */
    /* At this width there's no risk of collision between main column elements and asides so the float no longer necessary. */

@media screen and (min-width: 976px)
.ad-box--bigbox {
    float: none;
    clear: none;
    margin-top: 2em;
}
```

Apparently we do still need this float on the ad-box to prevent collisions with xl and xxl figures (which must be floated left to prevent huge gaps in the text column).

Enabling the ad-box left float again does seem to solve this layout problem. But it will mean the positioning of the ad blocks will become a bit more variable because it will shift depending on the presence of other floating objects. That said, I've tested it on 8 or 9 different pages and out of those 16–18 ads only 3 actually changed position at all and never by more than about 125 px.

It's pushed to develop and ready for deployment: 97f7b7f4893d8dba2bcba2d9ac7298f09e8673d4

Please keep an eye open for any other ad float oddities.

**Comment by MrBryan** [comment]
I pulled  - it is still breaking up paragraphs: 

![image](https://cloud.githubusercontent.com/assets/1721891/17485858/eb4652f2-5d5c-11e6-8dfc-da83d06ec625.png)

**Comment by alexgreen** [comment]
So it seems like we have 2 problems that may have mutually exclusive solutions:
- Aside.ad-boxes must float left to prevent collision with adjacent figures (see screenshot at top of this thread).
- Aside.ad-boxes must float:none to prevent breaking paragraphs.

Given these options, I think breaking paragraphs is the lesser of 2 evils.

One other option I haven't tried yet is to put the ad-boxes inside a figure instead of an aside. I could try that but it's unlikely to produce different results.

**Comment by phillipadsmith** [comment]
> One other option I haven't tried yet is to put the ad-boxes inside a figure instead of an aside. I could try that but it's unlikely to produce different results.

Any luck, @alexgreen

**Comment by alexgreen** [comment]
The fix b5f1b3af38ce0d24ed90d5f66a3ab10778e39e7a from #349 should fix this issue too

**Comment by MrBryan** [comment]
deployed
